src/test/java/bit/javaoop/BudgetManagerTest.java

```diff
@@ -0,0 +1,58 @@
+package bit.javaoop;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public class BudgetManagerTest {
+    private Manager manager1,manager2;
+    @BeforeEach
+    void setUp(){
+        manager1=new FixedBudgedManager("",15001,20001);
+        manager2=new FixedBudgedManager("",22001,20001);
+
+    }
+    @Test
+    void shouldHireEmployee(){
+        //given
+        AbstractEmployee e1=new Employee("",100);
+        AbstractEmployee e2=new Employee("",2_000);
+        AbstractEmployee e3=new Employee("",20_000);
+        //when
+        manager1.hire(e1);
+        manager1.hire(e2);
+        manager2.hire(e1);
+        manager2.hire(e2);
+        manager2.hire(e3);
+        //then
+        assertEquals(2,manager1.getEmployees_list().size());
+        assertEquals(2,manager2.getEmployees_list().size());
```

Comment:
- Dodałbym enterki przed blokami //given //when //then ładniej to wtedy wygląda:
![image](https://user-images.githubusercontent.com/18233796/38471966-d7eae58c-3b78-11e8-9a7b-84fa4e68425a.png)
- I generalnie wcisnąłeś tutaj dwie klasy abstrakcji ( przypadki testowe ) do jednego testu powinno to zostać rozbite na dwa testy a testy nazywać się np. shouldHireAllEmployees i shouldNotHireEmployeeOverBudget